### DISCUSSION TITLE
New Install - No Config Option Available

### DISCUSSION THREAD

**Discussion by JTP335d** [hassio-addons/addon-ssh#540] [Q&A] [answered]
I have installed this add-on onto a fresh haOS install on an SD card in a raspberry pi. I have never used this add-on before, so have no previous experience.  The config tab does not give me an option to edit/add ssh configuration. Switching to yaml mode does nothing either. The app will not start (obviously) because the config is incomplete. Am I missing something?

Tried: rebooting OS, rebooting RPi, removing and reinstalling

This is a test platform for a (new)project and has no other apps/integrations installed yet.

![Screen Shot 2023-04-20 at 12 52 13 PM](https://user-images.githubusercontent.com/77040947/233460880-9664769c-394d-40ec-bd95-d359d45d5fe0.png)

**Comment by frenck** [comment]
This is not the issue tracker....

Anything in the logs?

**Reply by JTP335d** [reply]
Not sure if this is an issue...or if I missed something basic. Have never had this problem with an add-on before. 

The only thing in the logs is the add-on didn't start. (because config is incomplete)

**Reply by frenck** [reply]
> The only thing in the logs is the add-on didn't start. (because config is incomplete)

Cool... care to share what is exactly in there?

../Frenck

**Reply by JTP335d** [reply, marked as answer]
Thanks for the quick reply. 

I just tried installing it on my main instance. Same thing. I'm using Safari, so I tried from Firefox and there the config editor shows up properly. Browser issue.

**Reply by frenck** [reply]
Please report that @ https://github.com/home-assistant/frontend/issues

As that is something they should look at.

../Frenck

**Comment by JTP335d** [comment]
Thank You.
Reported.
Jason
